src/components/QuickLook.js

```diff
@@ -156,7 +161,7 @@ function RecentOrders() {
         <>
           <ul className="yst-flex-1">
             {orders.data?.map((order) => (
-              <Card as="li" key={order.id}>
+              <Card as="a" key={order.id} href={RuntimeSdk.adminUrl(individualOrderLink(order.id))}>
```

Comment:
Can we check if the api response itself has any information on the link to the post?

Reply:
I can't find ay redirect link in the response